src/langchain_google_spanner/vector_store.py

```diff
@@ -785,6 +785,8 @@ def delete(
 
                 values.append(value)
 
+        delete_row_count = 0
```

Comment:
Was moving this outside the function intentional?

Reply:
Yes, to use it outside the block.